cardano-sl.cabal

```diff
@@ -30,6 +30,12 @@ Flag with-explorer
 
   description: Build with explorer
 
+flag bench
```

Comment:
I checked latest logs from CI: no more criterion. But first PR against time-warp-nt should be merged, then I still want it to be tried whether we can get rid of this `flag bench`.  Maybe after all changes we can simply not pass `--bench` to build.